repository/mysql/queue.go

```diff
@@ -38,12 +44,19 @@ func (r *queueRepository) Add(q *model.Queue) error {
 			max_dispatches_per_second = VALUES(max_dispatches_per_second),
 			max_burst_size = VALUES(max_burst_size)
 	`
-	_, err = r.db.Exec(sql, q.Name, q.MaxDispatchesPerSecond, q.MaxBurstSize)
+	res, err = r.db.Exec(sql, q.Name, q.MaxDispatchesPerSecond, q.MaxBurstSize)
 	if err != nil {
-		return err
+		return updated, err
+	}
+	i, err = res.RowsAffected()
+	if err == nil {
+		updated = updated || (i != 0)
 	}
 
-	return r.updateRevision()
+	if updated {
+		return updated, r.updateRevision()
```

Comment:
When `updateRevision()` is called, it is [captured by `configWatcher`](https://github.com/fireworq/fireworq/blob/66433271014e31587136956ee3535a1bc6ca40a8/service/config_watcher.go#L42), which is [initialized by a `Service`](https://github.com/fireworq/fireworq/blob/66433271014e31587136956ee3535a1bc6ca40a8/service/service.go#L45-L52).